src/model/metric.py

```diff
@@ -1,5 +1,15 @@
+import os
 import torch
 from abc import abstractmethod
+import tempfile
+
+import numpy as np
+from torchvision import transforms
+
+from utils.util import UnNormalize, lib_path, import_given_path
+
+inception = import_given_path("inception", os.path.join(lib_path, 'pytorch_fid/inception.py'))
```

Comment:
Why do we need to import the lib like this?

Reply:
Because I don't want to change `sys.path` directly